cmd/node/config/economics.toml

```diff
@@ -1,7 +1,20 @@
 #Economics config of the node
 [GlobalSettings]
-    TotalSupply = "20000000000000000000000000000" #20BILERD
-    MinimumInflation = 0.0
+    GenesisTotalSupply = "20000000000000000000000000" #20BILERD
+    MinimumInflation   = 0.0
+    YearSettings = [
+        {Year = 1, MaximumInflation = 10.845},
```

Comment:
Maybe switch from year = 1 to year = 2020? I think this will be clearer

Reply:
it will be hard to compute. it will depend on the machines time when we do the first setup. The year starts with Elrond :). and it is not for 2020, it is for first year from genesis start 12 months.